include/kernels/dslash_staggered.cuh

```diff
@@ -104,13 +103,18 @@ namespace quda
         if (doHalo<kernel_type>(d) && ghost) {
           const int ghost_idx = ghostFaceIndexStaggered<1>(coord, arg.dim, d, 1);
           const Link U = arg.improved ? arg.U(d, coord.x_cb, parity) : arg.U(d, coord.x_cb, parity, StaggeredPhase(coord, d, +1, arg));
-          Vector in = arg.halo.Ghost(d, 1, ghost_idx + src_idx * arg.nFace * arg.dc.ghostFaceCB[d], their_spinor_parity);
-          out = mv_add(U, in, out);
+          for (auto s = 0; s < n_src_tile; s++) {
```

Comment:
Should `#pragma unroll` be added here? Although I would assume the compiler does this by itself already.

Reply:
Probably a good idea.  I'll add that on next push.

Reply:
[22393c5](https://github.com/lattice/quda/pull/1515/commits/22393c5a893b20a99b85b1f92dc86e32ef1c1d40)